src/primitives/ray.jl

```diff
@@ -49,4 +49,5 @@ direction(r::Ray) = r.v
 
 Base.in(p::Point, r::Ray) = p ∈ Line(r.p, r.p + r.v) && (p - r.p) ⋅ r.v ≥ 0
 
+# Decision: use isapprox 
 ==(r₁::Ray, r₂::Ray) = (r₁.p == r₂.p) && (r₁.p + r₁.v) ∈ r₂ && (r₂.p + r₂.v) ∈ r₁
```

Comment:
Decision: use `isapprox`?

Reply:
Yes, it would be great to use isapprox explicitly here.